doc/quick-start.rst

```diff
@@ -59,6 +59,37 @@ And build it with:
 
 The executable will be built as ``_build/default/hello_world.exe``
 
+Building a hello world program using Core and Jane Street PPXs
+==============================================================
+
+Write this jbuild:
+
+.. code:: scheme
+
+    (jbuild_version 1)
+    
+    (executables
+     ((names (hello_world))
+      (libraries (core))
+      (preprocess (pps (ppx_jane ppx_driver.runner)))
+     )
+     
+This ``hello_world.ml`` file:
+
```

Comment:
Fixed now. the Lwt example above is incorrectly in scheme, but I guess that's a separate PR.